BetterMD/__init__.py

```
@@ -1,10 +1,48 @@
+        import requests as r
+        text = r.get(url).text
+
+        if text.startswith("<!DOCTYPE html>"):
+            text = text[15:]
+
+        return Symbol.from_html(text)
+
+class MD:
+    @staticmethod
+    def from_string(md:'str'):
+        return Symbol.from_md(md)
```

Ruff (0.8.2)

`Symbol` may be undefined, or defined from star imports

(F405)